source/_data/glossary.yml

```diff
@@ -31,7 +33,11 @@
 - topic: hass
   description: "HASS or [hass](/docs/tools/hass/) is often used as an abbreviation for Home Assistant. It is also the command-line tool."
 - topic: Hass.io
-  description: "[Hass.io](/hassio/) is an operating system that will take care of installing and updating Home Assistant, is managed from the Home Assistant UI, allows creating/restoring snapshots of your configuration, and can easily be extended."
+  description: "[Hass.io](/hassio/) is an full UI managed home automation ecosystem that runs Home Assistant, the Hass.io Supervisor and add-ons. It comes pre-installed on HassOS, but can be installed on any Linux system. It leverages Docker, which is managed by the Hass.io Supervisor."
+- topic: Hass.io Supervisor
+  description: "The Hass.io Supervisor is a program that manages the Hass.io ecosystem, taking care of installing and updating Home Assistant, add-ons, itself and, if used, updating your HassOS operating system."
```

Comment:
"the Hass.io ecosystem" -> "your Hass.io installation" ?

Reply:
I avoided `your`, because we cannot assume the reader is using it.

Reply:
Adjusted to:

> The Hass.io Supervisor is a program that manages a Hass.io installation, taking care of installing and updating Home Assistant, add-ons, itself and, if used, updating the HassOS operating system.